etk-analyze/src/sym.rs

```diff
@@ -764,6 +782,8 @@ mod z3_visit {
                 Sym::Difficulty => BV::new_const(self.context, "difficulty", 256),
                 Sym::GasLimit => BV::new_const(self.context, "gaslimit", 256),
                 Sym::ChainId => BV::new_const(self.context, "chainid", 256),
+                Sym::SelfBalance => BV::new_const(self.context, "selfbalance", 256),
```

Comment:
This needs to be a `fresh_const`, since the value can change inside of a single frame of execution.

Reply:
ah! fascinating